src/schema/suffixes/eeg.yaml

```diff
@@ -0,0 +1,4 @@
+---
+name: Electroencephalography
+description: |
+  Electroencephalography recording data.
```

Comment:
Am I right in assuming that some of the created YAML files are not yet integrated in the spec text in this PR?

Reply:
I think the only part of the specification with suffix tables is the MRI page, so none of the other suffixes, including `eeg`, are rendered in the text. To do so, we'd need to introduce new tables rather than just replace existing ones, I believe.

Reply:
:+1: